keyboards/eggman/readme.md

```diff
@@ -0,0 +1,19 @@
+# eggman
+
+![eggman](https://i.imgur.com/x4kaGml.jpg)
```

Comment:
```suggestion
![eggman](https://i.imgur.com/x4kaGmll.jpg)
```

Reply:
change committed, thanks for the review faux